package.json

```diff
@@ -1,35 +1,46 @@
 {
-  "name": "nextjs-template",
-  "private": true,
-  "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
-    "lint": "next lint"
-  },
-  "dependencies": {
-    "next": "12.1.6",
-    "react": "18.1.0",
-    "react-dom": "18.1.0"
-  },
-  "devDependencies": {
-    "@commitlint/cli": "^17.0.2",
-    "@commitlint/config-angular": "^17.0.0",
-    "@sapphire/eslint-config": "^4.3.6",
-    "@sapphire/prettier-config": "^1.4.3",
-    "@sapphire/ts-config": "^3.3.4",
-    "@types/node": "17.0.8",
-    "@types/react": "18.0.12",
-    "@typescript-eslint/eslint-plugin": "^5.27.1",
-    "@typescript-eslint/parser": "^5.27.1",
-    "eslint": "^8.17.0",
-    "eslint-config-next": "12.1.6",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "husky": "^8.0.1",
-    "lint-staged": "^13.0.1",
-    "sass": "^1.52.3",
-    "typescript": "4.7.3"
-  },
-  "packageManager": "yarn@3.2.1"
+	"name": "nextjs-template",
+	"version": "3.0.0",
+	"main": "dist/src/server",
+	"private": true,
+	"scripts": {
+		"dev": "NODE_ENV=development ts-node src/server",
+		"build": "yarn build:server && yarn build:app",
+		"build:server": "tsc --project src/server --outDir ./dist",
+		"build:app": "next build",
+		"start": "next start",
+		"lint": "next lint"
+	},
+	"dependencies": {
+		"@prisma/client": "^3.15.2",
+		"chokidar": "^3.5.3",
+		"dotenv": "^16.0.1",
+		"express": "^4.18.1",
+		"nanoid": "^4.0.0",
```

Comment:
TODO: Find `nanoid` alternative cuz ESM is annoying with TS
*@DaanGamesDG (so I don't forget)*

Reply:
Replace with [ShortUniqueId](https://www.npmjs.com/package/short-unique-id)